src/Compilers/CSharp/Portable/Symbols/Source/GlobalExpressionVariable.cs

```diff
@@ -27,7 +27,7 @@ internal class GlobalExpressionVariable : SourceMemberFieldSymbol
             : base(containingType, modifiers, name, syntax, location)
         {
             Debug.Assert(DeclaredAccessibility == Accessibility.Private);
-            _typeSyntax = typeSyntax.GetReference();
+            _typeSyntax = typeSyntax?.GetReference();
```

Comment:
>_typeSyntax [](start = 12, length = 11)

Consider renaming this field to ```_typeSyntaxOpt```

Reply:
Also consider adding a comment explaining when we don't have the ```typeSyntax```

---
In reply to: [156529109](https://github.com/dotnet/roslyn/pull/23209#discussion_r156529109) [](ancestors = 156529109)